examples/hierarchical_logistic_regression.py

```diff
@@ -69,21 +69,21 @@ def log_prob(self, xs, zs):
         log_prior = -self.prior_variance * tf.reduce_sum(zs*zs, 1)
         return log_lik + log_prior
 
-def build_toy_dataset(n_data=40, noise_std=0.1):
+def build_toy_dataset(n_minibatch=40, noise_std=0.1):
```

Comment:
This one refers to the number of data points for building the toy data set. So maybe something like `N`?